internal/tool/tool.go

```diff
@@ -193,6 +215,17 @@ func (t codacyTrivy) runVulnerabilityScanning(ctx context.Context, toolExecution
 	return mapIssuesWithoutLineNumber(filterIssuesFromKnownFiles(issues, *toolExecution.Files)), nil
 }
 
+// getSBOM produces a SBOM result from `report`.
+func (t codacyTrivy) getSBOM(ctx context.Context, report ptypes.Report) (codacy.SBOM, error) {
+	marshaler := cyclonedx.NewMarshaler("") // Trivy version
```

Comment:
We have a way of getting the Trivy version by looking at the `go.mod` file. 

Since `codacy-trivy`'s source is in the Docker image, I think we can call it here. I'll look into it.